src/Authenticator/JwtAuthenticator.php

```diff
@@ -90,25 +91,23 @@ public function authenticate(ServerRequestInterface $request, ResponseInterface
         $result = json_decode(json_encode($result), true);
 
         $key = IdentifierInterface::CREDENTIAL_JWT_SUBJECT;
-        if (empty($result[$key])) {
+        if (!array_key_exists($key, $result)) {
```

Comment:
Best to revert all array_key_exists code smells :)

Reply:
What's the problem in this case?

Reply:
We want to know if there is a value, not if a key exists that could even have null value.

Reply:
I'll wait for more comments to come in and answer all of them / fix all of them in one go.

Reply:
Revert back all `array_key_exists()` to `isset()`.